public/packs/manifest.json

```diff
@@ -4,10 +4,10 @@
   "assets/glyphicons-halflings-regular.ttf": "/packs/assets/glyphicons-halflings-regular.e18bbf611f2a2e43afc071aa2f4e1512.ttf",
   "assets/glyphicons-halflings-regular.woff": "/packs/assets/glyphicons-halflings-regular.fa2772327f55d8198301fdb8bcfc8158.woff",
   "assets/glyphicons-halflings-regular.woff2": "/packs/assets/glyphicons-halflings-regular.448c34a56d699c29117adc64c43affeb.woff2",
-  "dashboard.js": "/packs/dashboard-bf5a87fcb060e76c9b61.js",
-  "dashboard.js.map": "/packs/dashboard-bf5a87fcb060e76c9b61.js.map",
-  "navigation.js": "/packs/navigation-c4c6613e2ce14fb63092.js",
-  "navigation.js.map": "/packs/navigation-c4c6613e2ce14fb63092.js.map",
-  "policies.js": "/packs/policies-221156de03fef790b50b.js",
-  "policies.js.map": "/packs/policies-221156de03fef790b50b.js.map"
+  "dashboard.js": "/packs/dashboard-1d1f099fe0e7e0ceba6b.js",
```

Comment:
Are we sure we are modifying the manifest?
@didierofrivia it looks like the packs were not saved correctly before

Reply:
I re-compiled Webpack and committed packs as well, waiting for @didierofrivia anyways.